src/load/azext_load/tests/latest/recordings/test_load_test_app_component.yaml

```diff
@@ -136,14 +136,15 @@ interactions:
       message: Not Found
 - request:
     body: '{"displayName": "CLI-Test", "description": "Test created from az load test
-      command", "keyvaultReferenceIdentityType": "SystemAssigned", "environmentVariables":
-      {"rps": 1, "duration_in_sec": "1"}, "secrets": {}, "certificate": null, "loadTestConfiguration":
-      {"engineInstances": 1, "quickStartTest": false, "splitAllCSVs": false}, "passFailCriteria":
-      {"passFailMetrics": {"34ea3df8-1400-49e8-99c5-93e6c12929d8": {"aggregate": "avg",
-      "clientMetric": "requests_per_sec", "condition": ">", "value": "78"}, "fd06c877-11c9-47c0-b021-65b46688a694":
-      {"aggregate": "percentage", "clientMetric": "error", "condition": ">", "value":
-      "50"}, "5ad522a8-a6a6-4631-9f24-3edc6513c0e1": {"aggregate": "avg", "clientMetric":
-      "latency", "condition": ">", "value": "200", "requestName": "GetCustomerDetails"}}}}'
+      command", "keyvaultReferenceIdentityType": "SystemAssigned", "publicIPDisabled":
```

Comment:
do we need to update all recordings. This is an optional param right. It might be sufficient to either add new or update few targetted tests to cover the spectrum of
publicIPDisabled: null, false, true for public and private tests?

Reply:
Have updated default config file to test all the arguments, this change has updated many test recording files. Updated all the test with the new recording which are passing, just to be on safer side.

Reply:
added more cases for true and publicIPDisabled during public test